### PR TITLE
gh-126133: Only use start year in PSF copyright, remove end years

### PR DESCRIPTION
**Pull request by hugovk**
For https://github.com/python/cpython/issues/126133.

* Issue: gh-126133

----
📚 Documentation preview 📚: https://cpython-previews--126236.org.readthedocs.build/

### CODE REVIEW

File: Lib/email/_parseaddr.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2002-2007 Python Software Foundation
```
Comment (Wulian233) [inline Lib/email/_parseaddr.py:1]: Why are some files dated differently, 2001 and 2002?

File: Lib/email/feedparser.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2004-2006 Python Software Foundation
+# Copyright (C) 2004 Python Software Foundation
```
Comment (Wulian233) [inline Lib/email/feedparser.py:1]: Also 2004

File: Lib/email/_parseaddr.py (hunk 1 above)
Comment (hugovk) [reply Lib/email/_parseaddr.py:1]: It's usually when the file was created.

File: Doc/copyright.rst
Comment (AA-Turner) [inline Doc/copyright.rst:1]: This case (where there's multiple consecutive entries) makes me slightly prefer the `-present` form, as we have [1991-1995], [1995-2000], [2001], and then 2001 again. Or, we could remove the end years from the old entries, but I assume that would be more difficult to do?

A

File: Lib/email/__init__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2001-2007 Python Software Foundation
+# Copyright (C) 2001 Python Software Foundation
```
Comment (AA-Turner) [inline Lib/email/__init__.py:1]: Could we remove the copyright comments in files where the sole holder is the PSF? The general licence would cover this, I think?

Comment (Wulian233) [reply Lib/email/__init__.py:1]: Me too
https://github.com/python/cpython/issues/126133#issuecomment-2447150671

**Review by vstinner — APPROVED** [review]
LGTM

File: Lib/email/__init__.py (hunk 3 above)
Comment (gpshead) [reply Lib/email/__init__.py:1]: my personal preference is to not remove statements that exist.  just get us out of the continually sometimes updating the year heck. :)

**Review by gpshead — APPROVED** [review]

**Pull request merged**